packages/manager/modules/email-domain/src/general-informations/email-domain-general-informations.html

```diff
@@ -60,6 +60,31 @@
                         >
                     </oui-tile-description>
                 </oui-tile-definition>
+                <oui-tile-definition
+                    data-term="{{ ::'email_tab_domain_diagnostic' | translate }}"
+                >
+                    <oui-tile-description>
+                        <div>
+                            <button
+                                type="button"
+                                data-ng-if="ctrlTabGeneralInformations.getSpfConfig() === 'ok' || ctrlTabGeneralInformations.getSpfConfig() === 'fail'"
```

Comment:
Can `spfConfig` change by any action ? If not, you should use one time binding for optimization`